src/css/proposal.css

```
@@ -1 +1,403 @@
+}
+
+.proposal-title {
+    max-width: 776px;
+    position: relative;
+    top: 40px;
+    font-size: 38px;
+    font-weight: 700;
+    line-height: 40px;
+    color: var(--secondary-color);
+}
+
+@media screen and (min-width: 768px) {
+    .proposal-title {
+        min-width: 640px;
+        font-size: 60px;
+        line-height: 70px;
+        top: 40px;
+        min-width: 640px;
+    }
+}
+
+@media screen and (min-width: 1280px) {
+    .proposal-title {
+        max-width: 776px;
+    }
+}
+
+.proposal-main-card-container {
+    z-index: 10;
+    display: grid;
+    align-items: start;
+    position: absolute;
+    background-color: var(--secondary-color);
+
+    grid-template-columns: 295px auto;
+    height: 408px;
+    border-radius: 16px;
+    margin-top: 96px;
+}
+
+@media screen and (min-width: 768px) {
+    .proposal-main-card-container {
+        grid-template-columns: 479px auto;
+        margin-top: 140px;
+        column-gap: 5px;
+        max-width: 668px;
+        margin-left: -14.07px;
+        height: 500px;
+        border-radius: 30px;
+    }
+}
+
+@media screen and (min-width: 1280px) {
+    .proposal-main-card-container {
+        grid-template-columns: 746px auto;
+        margin-top: 178px;
+        max-width: 1024px;
+        height: 471px;
+    }
+}
+
+.proposal-left-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-start;
+
+    min-height: 255px;
+}
+
+@media screen and (min-width: 768px) {
+    .proposal-left-container {
+        min-height: 604px;
+    }
+}
+
+@media screen and (min-width: 1280px) {
+    .proposal-left-container {
+        min-height: 604px;
+    }
+}
+
+.proposal-right-container {
+    display: none;
+}
+
+@media screen and (min-width: 768px) {
+    .proposal-right-container {
+        display: block;
+        margin-top: 200px;
+        margin-left: 0;
+    }
+}
+
+@media screen and (min-width: 1280px) {
+    .proposal-right-container {
+        margin-top: 112px;
+        margin-bottom: 112px;
+        margin-left: 72px;
+    }
+}
+
+.proposal-bg-mid-card { 
+    z-index: 0;
+    position: absolute;
+    background-color: var(--secondary-accent-color);
+
+    width: 295px;
+    height: 408px;
+    border-radius: 16px;
+    margin-top: 100px;
+    transform: rotate(-11deg);
+}
+
+@media screen and (min-width: 768px) {
+    .proposal-bg-mid-card {
+        width: 668px;
+        height: 500px;
+        border-radius: 30px;
+        margin-left: -20px;
+        margin-top: 160px;
+        transform: rotate(-9.8deg);
+    }
+}
+
+@media screen and (min-width: 1280px) {
+    .proposal-bg-mid-card {
+        width: 1050px;
+        height: 463px;
+        border-radius: 30px;
+        margin-left: -14px;
+        margin-top: 180px;
+        transform: rotate(-8deg);
+    }
+}
+
+.proposal-bg-bottom-card {
+    z-index: 5;
+    position: absolute;
+    opacity: 0.5;
+    background-color: var(--secondary-color);
+
+    width: 295px;
```

```suggestion
    background-color: var(--secondary-color);
    width: 295px;
```

and please don't just copy-paste code check re-writings